fastlane/Fastfile

```diff
@@ -428,6 +429,59 @@ platform :android do
     )
   end
 
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build do |options|
+      UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
+
+      gradle(
+          task: ":app:assembleRelease",
+          properties: {
+          "skipSentryProguardMappingUpload" => true
+        }
+      )
+
+      upload_path = upload_to_s3(
+          bucket: 'a8c-apps-public-artifacts',
+          key: "pocketcasts-mobile-prototype-build-#{generate_prototype_build_number}.apk",
+          file: lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH],
+          skip_if_exists: true
+      )
+
+      install_url = "#{PROTOTYPE_BUILD_DOMAIN}/#{upload_path}"
+      comment_body = prototype_build_details_comment(
+        app_display_name: "📱 Mobile",
+        download_url: install_url,
+        fold: true
+      )
+
+      unless ENV['BUILDKITE_PULL_REQUEST'].nil?
+          comment_on_pr(
+              project: 'automattic/pocket-casts-android',
+              pr_number: Integer(ENV['BUILDKITE_PULL_REQUEST']),
+              reuse_identifier: "app-prototype-build-link",
+              body: comment_body
+          )
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/9fb120d7cb9100f73700cfa1412673f9ce6dd3e1/fastlane/Fastfile#L463"></span>Layout/ClosingParenthesisIndentation: Indent <code>)</code> to column 12 (not 10)</td>
    </tr>
  </tbody>
</table>